lib/homepage_options/choose_option.dart

```diff
@@ -22,7 +22,7 @@ class ChooseOption extends StatelessWidget {
           onPressed: () {
             Navigator.push(

```

Comment:
```suggestion
            Navigator.pushReplacement(

```
Dan word de pagina vervangen door de home pagina docent ipv dat we meerdere paginas op elkaar beginnen stacken. Met bonus dat de backbutton verdwijnt dan uit de homepage docent. (en zo kunt uitloggen via een knop dan telkens terug te navigeren tot je bent uitgelogd)